### DISCUSSION TITLE
how to apply for Githuv education pack

### DISCUSSION THREAD

**Discussion by shaynzz** [community/community#37226] [GitHub Education] [answered]
please let me know

**Comment by fykaa** [comment, marked as answer]
1) Visit [GitHub Education](https://education.github.com/) and click Get benefits in the top-right corner.
2) Under “Which best describes your academic status?”, select Student.
3) Then, complete one of the following:
If you have a school-issued email, select (or add it).
Please note that email addresses must be verified and any addresses with plus signs (“+”) are not eligible.
4) If you don’t have a school-issued email, follow the prompts to fill out additional information. You’re still eligible even if you only have a personal Gmail address, as long as you can provide alternative documentation to verify your current student status.
You may be prompted for more information. When prompted, upload an image that demonstrates your academic status or confirms your enrollment.

5) If you do not have a dated student ID to upload, you may upload a current transcript, letter on school letterhead, or any documentation that demonstrates your current enrollment. Make sure the image is clear and easy to read. Other languages are acceptable but may take longer to translate.

6) Current transcripts or a letter from your school with a date are acceptable forms of valid dated documentation.
Enter your school’s name, share how you will use GitHub, and click Submit. We try to review and process applications within two weeks of submission.

**Comment by shaynzz** [comment]
Thanks
